master/internal/rm/kubernetesrm/spec.go

```diff
@@ -70,7 +70,10 @@ func (p *pod) configureResourcesRequirements() k8sV1.ResourceRequirements {
 			}
 		}
 	}
-	return k8sV1.ResourceRequirements{}
+	return k8sV1.ResourceRequirements{
```

Comment:
nit: move the return into the `default` case? so if someone adds another case, they have satisfy it, not break out and get these defaults